packages/eslint-plugin-orbit-components/docs/rules/arrow-functions-count.md

```diff
@@ -0,0 +1,34 @@
+# arrow-functions-count
+
```

Comment:
Using too many arrow functions in interpolations can have a negative impact on performance, because they have to be evaluated with execution context. This is done internally by wrapping all functions into `css` helper from `styled-components` . In most cases it's far more better to use one single arrow function and then use nested conditions that returns `css` if necessary.

Reply:
Use ☝️  as the description of this rule.